packages/block-library/src/image/editor.scss

```diff
@@ -24,19 +24,10 @@ figure.wp-block-image:not(.wp-block) {
 }
 
 // This is necessary for the editor resize handles to accurately work on a non-floated, non-resized, small image.
-.wp-block-image .components-resizable-box__container {
-	display: inline-block;
```

Comment:
@jorgefilipecosta do you remember  why you removed this?

Reply:
(For reference, this seems to have broken alignment of image blocks, see #22694 and #22693.)

Reply:
The PR seemed to remove the need for this code because the display of handles when selected was handled by the component, so it seemed this code was no longer necessary. I'm sorry for the trouble caused :(